src/cursor/common.rs

```diff
@@ -152,22 +153,20 @@ pub(crate) struct CursorSpecification {
 
 impl CursorSpecification {
     pub(crate) fn new(
```

Comment:
This refactor was in service of being able to pass an `at_cluster_time` field through easily, but even without that it seemed worthwhile to leave in.